src/test/scala/edu/ie3/simona/test/common/model/grid/SubGridGateMokka.scala

```diff
@@ -21,14 +21,14 @@ import org.scalatestplus.mockito.MockitoSugar
   */
 trait SubGridGateMokka extends MockitoSugar {
 
-  /** Mocks a node with it's basic needed information
+  /** Mocks a node with its basic needed information
     *
     * @param uuid
     *   Unique identifier of the node
     * @param subnet
     *   Sub net number
     * @return
-    *   [[NodeInput]] with these information
+    *   [[NodeInput]] with this information
```

Comment:
"information" can also mean plural, not necessarily wrong imho

Reply:
see above. But I'm not sure and I'll not insists to it.